Shared/Samples/Augment reality to navigate route/AugmentRealityToNavigateRouteView.swift

```diff
@@ -14,245 +14,243 @@
+        /// A Boolean value indicating whether a route stop is selected.
+        var didSelectRouteStop: Bool {
+            startPoint != nil || endPoint != nil
         }
-    }
-    
-    /// Monitors the asynchronous stream of tracking statuses.
-    ///
-    /// When new statuses are delivered, update the route's traversed and remaining graphics.
-    private func trackStatus() async {
-        guard let routeTracker = routeDataModel.routeTracker else { return }
-        for await status in routeTracker.$trackingStatus {
-            guard let status else { continue }
-            switch status.destinationStatus {
-            case .notReached, .approaching:
-                if let route = routeResult?.routes.first {
-                    let currentManeuver = route.directionManeuvers[status.currentManeuverIndex]
-                    statusText = currentManeuver.text
-                }
-            case .reached:
-                statusText = "You have arrived!"
-            @unknown default:
-                break
+        
+        /// A Boolean value indicating whether a route is selected.
+        var didSelectRoute: Bool {
+            startPoint != nil && endPoint != nil
+        }
+        
+        /// The view model for scene view in the sample.
+        let sceneModel = SceneModel()
+        
+        /// The status text displayed to the user.
+        @Published var statusText = ""
+        
+        deinit {
+            Task {
+                /// Stop the location data source.
+                await locationDisplay.dataSource.stop()
             }
         }
-    }
-}
-
-extension AugmentRealityToNavigateRouteView {
-    @MainActor
-    class RouteDataModel: ObservableObject {
-        /// An AVSpeechSynthesizer for text to speech.
-        let speechSynthesizer = AVSpeechSynthesizer()
-        /// The route task that solves the route using the online routing service, using API key authentication.
-        let routeTask = RouteTask(url: URL(string: "https://route-api.arcgis.com/arcgis/rest/services/World/Route/NAServer/Route_World")!)
-        /// The parameters for route task to solve a route.
-        var routeParameters = RouteParameters()
-        /// The route tracker.
-        @Published var routeTracker: RouteTracker?
-        /// The route result.
-        @Published var routeResult: RouteResult?
         
-        /// Monitors the asynchronous stream of voice guidances.
-        func trackVoiceGuidance() async {
-            guard let routeTracker = routeTracker else { return }
-            for try await voiceGuidance in routeTracker.voiceGuidances {
-                speechSynthesizer.stopSpeaking(at: .word)
-                speechSynthesizer.speak(AVSpeechUtterance(string: voiceGuidance.text))
+        /// Performs important tasks including setting up the location display, creating route parameters,
+        /// and loading the scene elevation source.
+        func setup() async throws {
```

Comment:
```suggestion
        func setUp() async throws {
```

Set up is an action here so it is two words, like [XCTest.setUp()](https://developer.apple.com/documentation/xctest/xctest/1500341-setup).